### PR TITLE
CLDR-18047 Update spec to final

### PR DESCRIPTION
**Pull request by pedberg-icu**
CLDR-18047

- [x] This PR completes the ticket.

Update spec to final. Remove the "Tech Preview" from title of the keyboards part.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by AEApple** [comment]
Can you give me two hours? I want to review all the spec tickets that are now in review and make sure we're not missing anything before we merge this one. Thanks!

File: docs/ldml/tr35-keyboards.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-## Unicode Technical Standard #35 Tech Preview
+## Unicode Technical Standard #35
```
Comment (srl295) [inline docs/ldml/tr35-keyboards.md:1]: thanks for catching this. I know i removed other wording but obviously missed this.

**Review by srl295 — DISMISSED** [review]

**Comment by pedberg-icu** [comment]
> Can you give me two hours? I want to review all the spec tickets that are now in review and make sure we're not missing anything before we merge this one. Thanks!

Sure, I will wait to merge until after 2:30 pm Pacific

**Comment by AEApple** [comment]
I've been searching and I still haven't been able to find @srl295 's final link checking of the spec in the tickets. The closest I have found is https://unicode-org.atlassian.net/issues/CLDR-18031 but it doesn't specifically list out link checking as part of the work.

Can someone please help me either verify we have completed the link checking for the spec before we merge this change? Thanks!

**Comment by srl295** [comment]
Yes, I have completed it

File: docs/ldml/tr35-keyboards.md [hunk 2]
```diff
@@ -16,12 +16,12 @@ This is a partial document, describing keyboards. For the other parts of the LDM
...
 
-_This is a draft document which may be updated, replaced, or superseded by other documents at any time.
+<!-- __This is a draft document which may be updated, replaced, or superseded by other documents at any time.
 Publication does not imply endorsement by the Unicode Consortium.
-This is not a stable document; it is inappropriate to cite this document as other than a work in progress._
+This is not a stable document; it is inappropriate to cite this document as other than a work in progress._ -->
 
-<!-- _This document has been reviewed by Unicode members and other interested parties, and has been approved for publication by the Unicode Consortium.
-This is a stable document and may be used as reference material or cited as a normative reference by other specifications._ -->
+This document has been reviewed by Unicode members and other interested parties, and has been approved for publication by the Unicode Consortium.
```
Comment (AEApple) [inline docs/ldml/tr35-keyboards.md:23]: Looks like it's missing an underscore?

**Review by AEApple — CHANGES_REQUESTED** [review]
Can we fix the underscore formatting issue I found?

File: docs/ldml/tr35-keyboards.md (hunk 2 above)
Comment (pedberg-icu) [reply docs/ldml/tr35-keyboards.md]: Thanks. good catch, just fixed.

**Comment by pedberg-icu** [comment]
> Can we fix the underscore formatting issue I found?

@AEApple Thanks for catching that, I just fixed it

**Review by AEApple — APPROVED** [review]

**Pull request merged**